test/h265_sei_parser_unittest.cc

```diff
@@ -0,0 +1,45 @@
+/*
+ *  Copyright (c) Facebook, Inc. and its affiliates.
+ */
+
+#include "h265_sei_parser.h"
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include "h265_common.h"
+#include "rtc_base/arraysize.h"
+#include "rtc_base/bit_buffer.h"
+
+namespace h265nal {
+
+class H265SeiParserTest : public ::testing::Test {
+ public:
+  H265SeiParserTest() {}
+  ~H265SeiParserTest() override {}
+};
+
+TEST_F(H265SeiParserTest, TestUserDataRegisteredItuTT35Sei) {
+  // fuzzer::conv: data
+  const uint8_t buffer[] = {
```

Comment:
curious where you got this SEI

Reply:
Eventually I just found an h.264 sample that has some SEIs and attempted a software transcode with ffmpeg and libx265.  These User Data seis came along for the ride.  I inspected the h.264 and h.265 specs and there are a few seis that have the exact same structure between the codecs, user data is one of these.  Which may be one of the reasons it gets carried along on the transcode operation.  Most of the other seis that have exactly the same structure, I haven't yet been able to find on an h.264 sample and try to perform the same trick.  The list of seis that should be the same according to my research are as follows: filler_payload(3), 
user_data_registered_itu_t_t35(4), 
user_data_unregistered(5),  
progressive_refinement_segment_start(16),
progressive_refinement_segment_end(17), 
mastering_display_colour_volume(137), 
content_light_level_info(144), 
alternative_transfer_characteristics(147), 
ambient_viewing_environment(148), 
content_colour_volume(149), 
equirectangular_projection(150), 
cubemap_projection(151), 
sphere_rotation(154), 
omni_viewport(156), 
annotated_regions(202)